lib/vrf.c

```diff
@@ -1032,3 +1032,18 @@ vrf_id_t vrf_generate_id(void)
 
 	return ++vrf_id_local;
 }
+
+vrf_id_t vrf_to_id(struct vrf *vrf)
+{
+	return vrf ? vrf->vrf_id : VRF_UNKNOWN;
+}
+
+const char *vrf_to_name(struct vrf *vrf)
+{
+	return vrf ? vrf->name : "NIL";
+}
+
+void *vrf_to_vrf_info(struct vrf *vrf)
```

Comment:
There is already a vrf info lookup function right? vrf_info_lookup?  Why not use that?

Reply:
hard to merge it for now.
as this function is used only in zebra context, I will move ti to zebra.